public/js/machine.js

```diff
@@ -1 +1,28 @@
-$('#machine-table').tablesort();
+//$('#machine-table').tablesort();
```

Comment:
In general -- comments like this should be removed from your pull request. Remember -- this is git! All of the previous code is available in the history.